auth/forward.go

```diff
@@ -0,0 +1,58 @@
+package auth
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/containous/traefik/log"
+	"github.com/containous/traefik/types"
+	"github.com/stretchr/stew/objects"
+)
+
+// Forward the authentication to a external server
+func Forward(forward *types.Forward, w http.ResponseWriter, r *http.Request, next http.HandlerFunc) {
+
+	client := http.Client{}
+
+	forwardReq, _ := http.NewRequest("GET", forward.Address, nil)
```

Comment:
@emilevauge any idea of a good error handling for this case, is it OK to return an 5x error code?